motor_control/throttle/Button.ino

```diff
@@ -0,0 +1,39 @@
+#include "Button.h"
+
+Button::Button(int ft = 5):Throttle(){
```

Comment:
When you implement the constructor, you need to pass the analog pin that you're reading to the base `Throttle` class. See the summary under [this PR](https://github.com/Bruin-SuperMileage/EV/pull/12) for an example.